nursery/load-dotnet-assembly.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: load dotnet assembly
```

Comment:
final question before we merge, should we use .NET here instead of dotnet?

Reply:
I'd personally prefer `.NET` but then the filename would need to be `load-.net-assembly.yml`? which I don't like.

Reply:
i think we strip out special characters, so it would be `load-net-assembly.yml`.

we could special case `.NET` to `dotnet` in the linter filename check.

Reply:
also agree that `.NET` is the preferred term.

Reply:
Yes, the `.` is stripped so the rule file would be `load-net-assembly.yml`

Reply:
see https://github.com/mandiant/capa/pull/1068

Reply:
currently have two other affected rules:

```console
❯ python scripts/lint.py rules/
INFO:lint:successfully loaded 774 rules
INFO:lint:collecting potentially referenced samples
                                                                                                                     
compiled to the .NET platform                                                                                        
  FAIL: filename doesn't match the rule name: Rename rule file to match the rule name, expected: "compiled-to-the-dotnet-platform.yml", found: "compiled-to-the-net-platform.yml"
                                                                                                                     
                                                                                                                     
encrypt data using AES via .NET                                                                                      
  FAIL: filename doesn't match the rule name: Rename rule file to match the rule name, expected: "encrypt-data-using-aes-via-dotnet.yml", found: "encrypt-data-using-aes-via-net.yml"
                                                                                                                     
linting rule: linked against OpenSSL            : 100%|█████████████████████████| 774/774 [00:06<00:00, 117.52rule/s]
rules with FAIL:
  - compiled to the .NET platform
  - encrypt data using AES via .NET
```

Reply:
other affected rules fixed here: https://github.com/mandiant/capa-rules/pull/573